eslint-plugin-prettier.js

```diff
@@ -342,12 +337,31 @@ module.exports = {
           }
         }
 
+        if (prettier) {
+          prettier.clearConfigCache();
```

Comment:
Not too sure about this but it proved necessary when using vscode, otherwise changes to the `.prettierrc` were not getting picked up.

Reply:
This makes ESLint a lot slower. I have a partial fix here, but it would be good to figure out a better way to do this: https://github.com/prettier/eslint-plugin-prettier/pull/303